views/install.sh.erb

```diff
@@ -26,6 +26,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+download_timeout=10
```

Comment:
I'm more amenable to having the default be to have them disabled but optionally allow them.  That would keep the same behavior by default and then people who want to opt-in could use it when and if it solves a problem.